### PR TITLE
[FLOC-1596] Use Flocker version in Homebrew recipes for releases

### PR DESCRIPTION
**Pull request by jongiddy**
The Homebrew recipe uses the Git commit SHA to make the class names work without mangling.

Since marketing, pre- and dev releases use simpler version numbering, we can use those directly.  This allows the Buildbot-generated recipe to be used for the release.

### CODE REVIEW

File: flocker_bb/builders/flocker.py [hunk 1]
```diff
@@ -525,25 +550,58 @@ def makeHomebrewRecipeTestFactory():
...
+
+    # Run VM from our home directory. This allows the staging branch to
+    # be run separately from the master branch.
+    factory.addStep(SetPropertiesFromEnv(
+        name='set-home',
+        description=['setting', "'HOME'", "property"],
+        descriptionDone=['set', "'HOME'", "property"],
+        variables=['HOME']))
+
+    # Getting the VM IP address from VMWare is difficult when we don't
```
Comment (adamtheturtle) [inline flocker_bb/builders/flocker.py]: Explain how to do it with a password.

File: flocker_bb/builders/flocker.py [hunk 2]
```diff
@@ -525,25 +550,58 @@ def makeHomebrewRecipeTestFactory():
...
+    # have a user password.  Currently, we just hardwire the two known
```
Comment (adamtheturtle) [inline flocker_bb/builders/flocker.py]: Document how these were set to these IPs (can be in the Google Drive doc [too])

**Comment by jongiddy** [comment]
Move to sub-task FLOC-1625

**Comment by jongiddy** [comment]
Moved to new JIRA sub-task, now PR #71

**Pull request closed without merging**